Classes/EventStore/EventStreamFilter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+
+class EventStreamFilter
```

Comment:
it's just a dummy really to make it working, I'll prepare a separate PR for the filter